### PR TITLE
build(deps): bump the go group across 1 directory with 6 updates

### PR DESCRIPTION
**Pull request by jachym-tousek-keboola**
Replaces #2166

### CODE REVIEW

**Review by hosekpeter — APPROVED** [review]
LGTM

File: internal/pkg/service/common/configmap/bind_test.go [hunk 1]
```diff
@@ -99,7 +99,6 @@ func TestGenerateAndBind_DefaultValues(t *testing.T) {
 
 	// TestConfig is unmodified, default values are kept
 	require.NoError(t, GenerateAndBind(cfg, &target))
-	assert.NotSame(t, expected, target)
```
[the remark this answers is not in the document]
Comment (Matovidlo) [reply internal/pkg/service/common/configmap/bind_test.go:102]: I would expect to have some deep equality of expected and target. Should I look at that?

**Review by Matovidlo — COMMENTED** [review]
Should I take a look at the configmap test?

File: internal/pkg/service/common/configmap/bind_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jachym-tousek-keboola) [reply internal/pkg/service/common/configmap/bind_test.go:102]: What do you mean? There is `assert.Equal(t, expected, target)` one line below?

**Review by Matovidlo — APPROVED** [review]

**Pull request merged**